data/data/manifests/openshift/baremetal-provisioning-config.yaml.template

```diff
@@ -9,4 +9,5 @@ spec:
   provisioningNetwork: "{{.Baremetal.ProvisioningNetwork}}"
   provisioningDHCPRange: "{{.Baremetal.ProvisioningDHCPRange}}"
   provisioningOSDownloadURL: "{{.ProvisioningOSDownloadURL}}"
+  additionalNTPServers: "[{{range $index, $server := .Baremetal.AdditionalNTPServers}}{{if $index}},{{end}}\"{{$server}}\"{{end}}]"
```

Comment:
This should be a list of strings (to match with the CRD in the CBO https://github.com/openshift/cluster-baremetal-operator/pull/449/files#diff-3199d649a76f2244e3cbe100488dc1ecab6e87dbca7efb6842da31ee63505f7aR152 )

I think this would do it
+  additionalNTPServers: [{{range $index, $server := .Baremetal.AdditionalNTPServers}}{{if $index}},{{end}}"{{$server}}"{{end}}]